pkg/virt-handler/non-root.go

```diff
@@ -46,30 +46,32 @@ func changeOwnershipAndRelabel(path string) error {
 }
 
 func changeOwnershipOfHostDisks(vmiWithAllPVCs *v1.VirtualMachineInstance, res isolation.IsolationResult) error {
-	for i := range vmiWithAllPVCs.Spec.Volumes {
-		if volumeSource := &vmiWithAllPVCs.Spec.Volumes[i].VolumeSource; volumeSource.HostDisk != nil {
-			volumeName := vmiWithAllPVCs.Spec.Volumes[i].Name
-			diskPath := hostdisk.GetMountedHostDiskPath(volumeName, volumeSource.HostDisk.Path)
-
-			_, err := os.Stat(diskPath)
-			if err != nil {
-				if os.IsNotExist(err) {
-					diskDir := hostdisk.GetMountedHostDiskDir(volumeName)
-					if err := changeOwnershipAndRelabel(filepath.Join(res.MountRoot(), diskDir)); err != nil {
-						return fmt.Errorf("Failed to change ownership of HostDisk dir %s, %s", volumeName, err)
-					}
-					continue
-				}
+	for _, volume := range vmiWithAllPVCs.Spec.Volumes {
+
+		if volume.VolumeSource.HostDisk == nil {
+			continue
+		}
+
+		volumeName := volume.Name
+		// This `hostdisk` I don't know where it's coming from.
```

Comment:
You mean this [from here](https://github.com/kubevirt/kubevirt/blob/9b1ac82e6f86f744a954d2c09965cb8a004106b7/pkg/host-disk/host-disk.go#L169), right?

If you agree would you remind removing this comment?